### PR TITLE
Alternative index

### PR DESCRIPTION
**Pull request by pmuir**
DO NOT MERGE

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 0319893eb88d7517519af2f7416f7073003ff0f8
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8e91584246c0174755cda3ea54e3c549f0e7fd83
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: config/webpack.common.js [hunk 1]
```diff
@@ -338,7 +338,8 @@ module.exports = function (options) {
         template: 'src/index.html',
         title: METADATA.title,
         chunksSortMode: 'dependency',
-        metadata: METADATA
+        metadata: METADATA,
+        filename: (isProd ? 'f8-ui.html' : 'index.html')
```
Comment (joshuawilson) [inline config/webpack.common.js]: interesting idea

**Comment by pmuir** [comment]
This allows us to implement the path based routing for the landing page vs
this page. KB and I are still working on it.

**Comment by pmuir** [comment]
This is working, we can merge when @kbsingh has the routing fixed up

**Pull request closed without merging**